src/App.css

```diff
@@ -37,6 +37,10 @@
   src: local("Monaco"), url(./styles/fonts/Monaco/Monaco.ttf) format("truetype");
 }
 
+*:focus {
```

Comment:
why do we change this?

Reply:
The browser puts these incredibly ugly outlines over buttons by default (for accessibility) and it misrepresents the intent of our styling